packages/rs-drive/src/drive/verify/state_transition/verify_state_transition_was_executed_with_proof/v0/mod.rs

```diff
@@ -198,6 +205,29 @@ impl Drive {
                             )])),
                         ))
                     }
+                    DocumentTransition::UpdatePrice(update_price_transition) => {
+                        let document = document.ok_or(Error::Proof(ProofError::IncorrectProof(format!("proof did not contain document with id {} expected to exist because of state transition (update price)", update_price_transition.base().id()))))?;
+                        let new_document_price : Credits = document.properties().get_integer(PRICE).map_err(|e| Error::Proof(ProofError::IncorrectProof(format!("proof did not contain a document that contained a price field with id {} expected to exist because of state transition (update price): {}", update_price_transition.base().id(), e))))?;
+                        if new_document_price == update_price_transition.price() {
```

Comment:
Should be `new_document_price != update_price_transition.price()`?

Reply:
Indeed